### PR TITLE
OCPBUGS-17428: workload info gatherer, add external image repo

### PR DESCRIPTION
**Pull request by ncaak**
This PR implements a new data enhancement to [Workload info gatherer](https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/workloads/gather_workloads_info.go) to add image repository to the archive. The repository image is only collected if it's from outside of Red Hat domain. 

## Categories

- [ ] Bugfix
- [X] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample Archive

- `docs/insights-archive-sample/config/workload_info.json`

## Documentation

- `docs/gathered-data.md`

## Unit Tests

- `pkg/gatherers/workloads/gather_workloads_info_test.go`

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Changelog

No

## Breaking Changes

No

## References

https://issues.redhat.com/browse/CCXDEV-11505
https://issues.redhat.com/browse/OCPBUGS-17428

### CODE REVIEW

File: docs/insights-archive-sample/config/workload_info.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "sha256:1bd85e07834910cad1fda7967cfd86ada69377ba0baf875683e7739d8de6d1b0"
       ],
       "firstCommand": "icTsn2s_EIax",
-      "firstArg": "2v1NneeWoS_9"
+      "firstArg": "2v1NneeWoS_9",
+      "repository":"quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a16c19e8f7372bec6e2b62f1eae6d4796d6aec8435679c7821fdc267fa667140"
```
Comment (tremes) [inline docs/insights-archive-sample/config/workload_info.json:26]: This must be hashed similarly as other values please.

Comment (ncaak) [reply docs/insights-archive-sample/config/workload_info.json:26]: - [ ]  Detect where in the code the values of the other properties are being hashed. Then hash this value as well.
- [ ] Update sample file

**Comment by tremes** [comment]
@ncaak I guess we first need to merge https://github.com/openshift/insights-operator/pull/812 for the testsuite to pass....sorry for the incovenience

**Comment by tremes** [comment]
I didn't try it, but looks good.
/lgtm

**Comment by tremes** [comment]
@ncaak You need to use the OCPBUGS in the commit title, if you want to backport

**Comment by tremes** [comment]
Update of existing gatherer. 
/override ci/prow/e2e-metal-ipi-ovn-ipv6
/override ci/prow/e2e-gcp-ovn-techpreview

**Pull request merged**